tests/test_plugins.py

```diff
@@ -0,0 +1,141 @@
+import tempfile
+import strax
+import straxen
+import numpy as np
+from immutabledict import immutabledict
+from strax.testutils import run_id, recs_per_chunk
+
+# Number of chunks for the dummy raw records we are writing here
+N_CHUNKS = 2
+
+##
+# Tools
+##
+
+@strax.takes_config(
+    strax.Option('secret_time_offset', default=0, track=False)
+)
+class DummyRawRecords(strax.Plugin):
+    """
+    Provide dummy raw records for the mayor raw_record types
+    """
+    provides = ('raw_records', 'raw_records_he', 'raw_records_nv',)
+    parallel = 'process'
+    depends_on = tuple()
+    data_kind = immutabledict(zip(provides, provides))
+    rechunk_on_save = False
+    dtype = {p: strax.raw_record_dtype() for p in provides}
+
+    def source_finished(self):
+        return True
+
+    def is_ready(self, chunk_i):
+        return chunk_i < N_CHUNKS
+
+    def compute(self, chunk_i):
+        t0 = chunk_i + self.config['secret_time_offset']
+        if chunk_i < N_CHUNKS - 1:
+            r = np.zeros(recs_per_chunk, self.dtype['raw_records'])
+            r['time'] = t0
+            r['length'] = r['dt'] = 1
+            r['channel'] = np.arange(len(r))
+        else:
+            r = np.zeros(0, self.dtype['raw_records'])
+        res = {p: self.chunk(start=t0, end=t0 + 1, data=r, data_type=p)
+               for p in self.provides}
+        return res
+
+# Don't concern ourselves with rr_aqmon et cetera
+forbidden_plugins = tuple([p for p in
+                           straxen.daqreader.DAQReader.provides
+                           if p not in DummyRawRecords.provides])
+
+
+def _run_plugins(st,
+                 make_all=False,
+                 **proces_kwargs):
+    """
+    Try all plugins (except the DAQReader) for a given context (st) to see if
+    we can really push some (empty) data from it and don't have any nasty
+    problems like that we are referring to some non existant dali folder.
+    """
+
+    with tempfile.TemporaryDirectory() as temp_dir:
+        st.storage = [strax.DataDirectory(temp_dir)]
+
+        # As we use a temporary directory we should have a clean start
+        assert not st.is_stored(run_id, 'raw_records'), 'have RR???'
+
+        # Create event info
+        target = 'event_info'
+        st.make(run_id=run_id,
+                targets=target,
+                **proces_kwargs)
+
+        # The stuff should be there
+        assert st.is_stored(run_id, target), f'Could not make {target}'
+
+        # I'm only going to do this for nT because:

```

Comment:
I do not get this comment. You are calling this function in test_1T.

Reply:
I'm calling this fucntion always for 1T with this argument:
https://github.com/XENONnT/straxen/blob/8a5ea4034126ab3c7dcee4bde95fd7170798859e/tests/test_plugins.py#L121

Therefore the ```if make_all``` is always ``False`` for 1T